.travis.yml

```diff
@@ -33,20 +33,20 @@ before_install:
   # - export PYTHONPATH=$PYTHONPATH: ???
 
   # install freesasa
-  - conda install cython
-  - sudo apt-get install -y libxml2 libxml2-dev libjson0 libjson0-dev
-  - wget http://github.com/mittinatten/freesasa/releases/download/2.0.2/freesasa-2.0.2.tar.gz
-  - mkdir freesasa
-  - tar -xvf freesasa-2.0.2.tar.gz -C freesasa --strip-components=1
-  - cd freesasa
-  - ./configure --disable-xml --disable-json --enable-python-bindings --with-python=python3.6 CFLAGS="-fPIC" --prefix=$HOME
-  - make
-  - make install
-  - cd ../
+  # - conda install cython
+  # - sudo apt-get install -y libxml2 libxml2-dev libjson0 libjson0-dev
+  # - wget http://github.com/mittinatten/freesasa/releases/download/2.0.2/freesasa-2.0.2.tar.gz
+  # - mkdir freesasa
+  # - tar -xvf freesasa-2.0.2.tar.gz -C freesasa --strip-components=1
+  # - cd freesasa
+  # - ./configure --disable-xml --disable-json --enable-python-bindings --with-python=python3.6 CFLAGS="-fPIC" --prefix=$HOME
+  # - make
+  # - make install
+  # - cd ../
```

Comment:
if freesasa is pip installable remove all of that and put simply free sasa in the install requirements

Reply:
Yep, it is changed.